app/js/app.js

```diff
@@ -7,7 +7,20 @@ const BLOCKED_APPS = [
 
 const SYSTEM_APP = 'app://system.gaiamobile.org/manifest.webapp';
 
-window.addEventListener('DOMContentLoaded', () => {
+if (document.documentElement) {
```

Comment:
Since this is so fragile, and the add-ons API is unstable, we should comment this explaining what we're doing, as well as potential ways that it can break.